src/main/java/com/celements/web/service/WebUtilsService.java

```diff
@@ -1459,4 +1461,11 @@ public EntityType resolveEntityTypeForFullName(String fullName,
     return componentManager.lookupMap(role);
   }
 
+  private IDocumentParentsListerRole getDocumentParentsLister() {
+    if (docParentsLister == null) {
+      docParentsLister = Utils.getComponent(IDocumentParentsListerRole.class);
```

Comment:
I refactored the component to a class field and therefore placed the comment there:

/*
 * not loaded as requirement due to cyclic dependency
 */
IDocumentParentsListerRole docParentsLister;